react-js/src/service/scanbot-sdk-service.ts

```diff
@@ -71,7 +72,19 @@ export class ScanbotSdkService {
     public async createDocumentScanner(detectionCallback: any) {
         const config: DocumentScannerConfiguration = {
             onDocumentDetected: detectionCallback,
-            containerId: ScanbotSdkService.DOCUMENT_SCANNER_CONTAINER
+            containerId: ScanbotSdkService.DOCUMENT_SCANNER_CONTAINER,
+            text: {
+                hint: {
+                  OK: 'Dokument wird erfasst... Bitte bewegen Sie die Kamera nicht.',
+                  OK_SmallSize: 'Das Dokument ist zu klein. Versuchen Sie bitte näher heranzugehen.',
+                  OK_BadAngles: 'Dies ist ein schlechter Kamerawinkel. Halten Sie bitte das Gerät gerade über das Dokument.',
+                  OK_BadAspectRatio: 'Drehen Sie das Gerät seitlich, so dass das Dokument besser in den Bildschirm passt.',
+                  OK_OffCenter: 'Versuchen Sie bitte, das Gerät über die Mitte des Dokuments zu halten.',
+                  Error_NothingDetected: 'Bitte halten Sie das Gerät zum Scannen über ein Dokument.',
+                  Error_Brightness: 'Es ist zu dunkel, bitte sorgen Sie für ausreichende Beleuchtung.',
+                  Error_Noise: 'Bitte legen Sie das Dokument auf eine glatte Oberfläche.',
+                }
+            },
```

Comment:
I would make these properties non-optional in the SDK and use English versions for them in the public example project